test/jdk/javax/swing/JTableHeader/TableHeaderBorderPositionTest.java

```diff
@@ -65,64 +64,40 @@ private static void test() {
         Dimension size = header.getPreferredSize();
         header.setSize(size);
 
-        int w = (int)Math.ceil(SCALE * size.width);
-        int h = (int)Math.ceil(SCALE * size.height);
-
-        BufferedImage imgHeader = new BufferedImage(w, h,
+        BufferedImage bufferedImage = new BufferedImage(WIDTH, HEIGHT,
                 BufferedImage.TYPE_INT_RGB);
```

Comment:
Hm, you have to create the image large enough to contain the table after scaling it by `SCALE`, so you still need `w` and `h` or you could inline `WIDTH * SCALE` and `HEIGHT * SCALE` correspondingly.